### ISSUE TITLE
DFRobot SAINSMARTKS LCD cannot be used with MRAA subplatform

### ISSUE THREAD

**Issue by deadprogram** [intel-iot-devkit/upm#503]
I am unable to use the DFRobot SAINSMARTKS LCD display when using the MRAA Firmata subplatform, due to the pin naming convention of subplatforms (512 + the pin number) exceeding the allowed size of the params for the SAINSMART (uint8).

```
  File "/usr/lib64/python2.7/site-packages/upm/pyupm_i2clcd.py", line 1334, in __init__
    this = _pyupm_i2clcd.new_SAINSMARTKS(rs, enable, d0, d1, d2, d3, keypad)
NotImplementedError: Wrong number or type of arguments for overloaded function 'new_SAINSMARTKS'.
  Possible C/C++ prototypes are:
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t,uint8_t,uint8_t,uint8_t,uint8_t,uint8_t,uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t,uint8_t,uint8_t,uint8_t,uint8_t,uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t,uint8_t,uint8_t,uint8_t,uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t,uint8_t,uint8_t,uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t,uint8_t,uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t,uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS(uint8_t)
    upm::SAINSMARTKS::SAINSMARTKS()
```

**Comment by deadprogram** [comment]
I wonder if this is the same problem with https://github.com/intel-iot-devkit/upm/issues/498

**Comment by deadprogram** [comment]
The new UPM 1.0.2 that was just released corrects this problem for me. Thank you!

**Comment by Propanu** [comment]
Did this fix #498 for you as well?

**Comment by deadprogram** [comment]
Yes it did. Both of these issues can now be closed, thanks.
